app/components/whac-a-mole/game/interface/template.hbs

```diff
@@ -0,0 +1,45 @@
+<div class='text-center'>
```

Comment:
Try to create `<Shared::Button />` component which will have default `class`, `type` and use `...attributes` - https://guides.emberjs.com/release/components/component-arguments-and-html-attributes/#toc_html-attributes

Make sure we are able to edit for ex. `type`
Example:
```js
<Shared::Button 
  class="btn-success"
  @label="Start"
  @type="submit" // default is `button`
  @onClick={{this.onStart}}
/>
```

And replace all occurs buttons in whole app